### ISSUE TITLE
What does Pomade actually do?

### ISSUE THREAD

**Issue by monkpit** [sauce-archives/pomade#3]
I found this package by searching for python / selenium / page objects. I see references to page objects throughout the repo, but I don't see anything telling me what exactly Pomade does to help me create page objects or use them. In fact I can't really tell much at all, except from the readme, apparently this package makes it easy to run my tests on Sauce via Nose. Is there a base class here I can inherit to create a my page objects?

The only references to this package I can find are on github itself, so far. I had hoped to find a blog post or tutorial or something explaining things in a little more detail than the readme.

**Comment by siemiatj** [comment]
I think Berni is your man to go to. As far as I remember Pomade basically
added the custom functionality we now have in our
SeleniumTestCase/Selenium2TestCase objects to a page object.

On Tue, Mar 8, 2016 at 8:02 AM, Kyle Pittman notifications@github.com
wrote:

> I found this package by searching for python / selenium / page objects. I
> see references to page objects throughout the repo, but I don't see
> anything telling me what exactly Pomade does to help me create page objects
> or use them. In fact I can't really tell much at all, except from the
> readme, apparently this package makes it easy to run my tests on Sauce via
> Nose. Is there a base class here I can inherit to create a my page objects?
> 
> The only references to this package I can find are on github itself, so
> far. I had hoped to find a blog post or tutorial or something explaining
> things in a little more detail than the readme.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/saucelabs/pomade/issues/3.

## 

_Kuba Siemiątkowski Software Engineer_
Twitter: @sasklacz
GitHub: @siemiatj
Sauce Labs http://saucelabs.com/ | Sauce blog http://sauceio.com/

Talk to us anytime: https://twitter.com/saucelabs
https://www.linkedin.com/company/sauce-labs
https://plus.google.com/+SauceLabs_Official/posts
https://www.facebook.com/saucelabs

**Comment by christian-bromann** [comment]
However I don't think this project is maintained anymore.

**Comment by bernii** [comment]
@monkpit yeah this project is not actively maintained anymore. Pomade is basically a set of extracted selenium good practices (spinning asserts, page objects, helpers, custom assertions) that makes it easy to write well abstracted tests and run them on Sauce too. Unfortunately we do not have any blog posts or tutorials available at this time but I think it might be informative to go through the source code too se how abstractions are defined.
